extensions/cornerstone-dynamic-volume/src/components/LineChartViewport/LineChartViewport.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { LineChart } from '@ohif/ui';
+
+const LineChartViewport = ({ displaySets }) => {
+  const displaySet = displaySets[0];
+  const { axis: chartAxis, series: chartSeries } = displaySet.chartData;
```

Comment:
again not sure if series is DICOM series or what

Reply:
`chartData` does not contain any DICOM related data but only label and points.